### ISSUE TITLE
Could anybody provide me the  pretrained models?

### ISSUE THREAD

**Issue by MockerPeking** [NVlabs/few-shot-vid2vid#22]
especially the pose model?  my  graphic card  is not good enough to train data, tks a lot!
u could  mail    erickyi@pku.edu.cn

**Comment by Yilen-fan** [comment]
I also want a pose model 
and my mail is mengmengboy@126.com 
thanks

**Comment by ghost** [comment]
maybe we could try crowd training the pose model similar to what was done for the unreleased flickr landscape model for spade

https://github.com/NVlabs/SPADE/issues/17

**Comment by tbbjymm21** [comment]
I want a face model
and my mail is 921785514@qq.com
thanks very much

**Comment by mnataru** [comment]
Would also like to receive a pretrained pose model 🙏
artemmnad@gmail.com

**Comment by howardgriffin** [comment]
> I also want a pose model
> and my mail is [mengmengboy@126.com](mailto:mengmengboy@126.com)
> thanks
Have you received the pose model? I also want the pose model and my email is 1403679965@qq.com. Thank you!

**Comment by howardgriffin** [comment]
> I also want a pose model
> and my mail is [mengmengboy@126.com](mailto:mengmengboy@126.com)
> thanks

Have you received the pose model? I also want the pose model and my email is 1403679965@qq.com. Thank you!

**Comment by azizsiyaev** [comment]
Kindly asking to share Pretrained Pose Model, please. Email: azizsiyaev.ai@gmail.com

**Comment by taroorat** [comment]
fhjkjfh@live.com

**Comment by thrfdth** [comment]
Could you please give me pretrained pose model as well? 
My Email: qitolefreulich@gmail.com

**Comment by CarolinGao** [comment]
I want a face model
and my mail is 352175516@qq.com
thanks very much

**Comment by k4rth33k** [comment]
If anyone following this thread or the repo in general who has successfully reproduced the results on the pose data, please share the specs of the system and the parameters you used.
I would very much appreciate it if anyone can contact me regarding this or send me the pose model at _**kartheek2000mike@gmail.com**_. 
Thank you for your patience.

**Comment by ahmedelm** [comment]
Can someone plesae share the face model trained on the FaceForensics dataset ? 
My email is elgharib@mpi-inf.mpg.de

Thanks
--Mohamed Elgharib
http://people.mpi-inf.mpg.de/~elgharib/

**Comment by TrueCarry** [comment]
Can someone send me pose model please?
truecarry@gmail.com

**Comment by mohsin127** [comment]
Hey if someone can also send me the pretrained models for Pose and face. That would be great Thank! 
m.mohsin127@yahoo.com

**Comment by magdev2022** [comment]
I want a face model
and my email is kgh919@outlook.com.
Please help me.

**Comment by lorrp1** [comment]
could someone train it using public FFHQ? and publish it?

**Comment by lorrp1** [comment]
https://gradient.paperspace.com/free-gpu

if somone would like to start traning here a good idea.

**Comment by Devetec** [comment]
ahhh i need pose model
my email is thecodingcat@outlook.com
help

**Comment by DFGANDP** [comment]
Could you please give me pretrained pose model?
My Email: woj.dl.gan@gmail.com
Thanks!

**Comment by jigenji** [comment]
Could you give me pretrained face model pelase?
kawakami-shinji929@g.ecc.u-tokyo.ac.jp

**Comment by unoShin** [comment]
Could anybody share Pretrained Pose Model please? Email: yoon731@korea.ac.kr

**Comment by shiluooulihs** [comment]
Could anybody please give me a pretrained face model?
My Email: shiluooulihs@gmail.com
Thanks!

**Comment by chenruyi** [comment]
Could you give me a pretrained face model?
Email: [chenruyi0627@outlook.com](chenruyi0627@outlook.com)
Thanks

**Comment by haigege** [comment]
I want a face model
and my mail is 949435735@qq.com
thanks very much

**Comment by haigege** [comment]
haigege commented 29 minutes ago
I want a face model
and my mail is 949435735@qq.com
thanks very much

**Comment by haigege** [comment]
haigege commented 29 minutes ago
I want a face model
and my mail is 949435735@qq.com
thanks very much

**Comment by scarymrgrey** [comment]
Please, share face model with me. I'll be very grateful! imbamrgrey@gmail.com

**Comment by etudemin** [comment]
Could anyone share a pretrained face model with me?
My Email: d063010005@student.nsysu.edu.tw
Thank you very much!

**Comment by NiKS1804** [comment]
Could you give me pretrained face model pelase? My Email I'd : niks9798@gmail.com

**Comment by baymaxdarwin** [comment]
Could you please share a pre-trained pose model?
my email: baymax.darwin@gmail.com

**Comment by huochf** [comment]
Could you please share a pre-trained pose or face or street model?
my email: 1350133767@qq.com

**Comment by SwitcherPark** [comment]
Could you please share a pre-trained pose model? Plz......... My email: narciss123@naver.com

**Comment by SwitcherPark** [comment]
Could you please share a pre-trained pose model?
Plz.......!!!
My email: narciss123@naver.com

**Comment by AhmedHisham99** [comment]
I need a pre-trained pose model please .
ah.abolfadl@gmail.com

**Comment by xiefeiwhu** [comment]
Could you give me a pretrained face model?
My Email: xiefei@whu.edu.cn

**Comment by viggyr** [comment]
I would highly appreciate if you could share the pretrained pose model 

Email: vradhakrishn@umass.edu

**Comment by Sentero-esp12** [comment]
Could someone please share pretrained face and pose models? I want to use them for animating artbreeder characters. My email is ernest.de.santes@gmail.com 
Thank you!

**Comment by blackuti** [comment]
hello, thank you for your time, would you help share the pretrained face model? thank you so much!
blackuti@gmail.com

**Comment by viggyr** [comment]
This is of no use. The model was never shared

**Comment by DuyguSerbes** [comment]
hello, thank you for your time, would you help share the pretrained face model? thank you so much! duygu_serbes@hotmail.com

**Comment by Olivia-Ye** [comment]
I'll be thankful if anyone could share me a dance model please. Thanks a lot!  956947802@qq.com  or  yeruiying13@gmail.com

**Comment by NeaRBrotheR** [comment]
I need a pre-trained pose and face model please .Thanks a lot!
My email is andy861117@gmail.com

**Comment by APeiZou** [comment]
@ak9250  @amnats  @tcwang0509   Hey if someone can also send me the pretrained models for Pose and face. That would be great Thank! My email is 875341218@qq.com

**Comment by Loc-VD** [comment]
I really want to try to the pose model, It would be great if you can share the pre-trained one for the pose at voducloc@gmail.com

**Comment by borishanzju** [comment]
I also want a pose model
and my mail is borishan815@zju.edu.cn
thanks

**Comment by borishanzju** [comment]
I also want a pose model
and my mail is borishan815@zju.edu.cn
thanks

**Comment by leepms** [comment]
I also want a street model
and my email is zhongqi.lin@zju.edu.cn
thanks.
